Back/Reciplastk.Gateway/Models/RolViewModel.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Reciplastk.Gateway.Models
+{
+    public class RolViewModel
```

Comment:
Class name is incorrect. 
```suggestion
    public class RoleViewModel
```